src/lib/OS/Process.cc

```diff
@@ -274,10 +274,11 @@ uint64_t Process::createStack(uint64_t stackStart) {
     stringBytes.push_back(0);
   }
   // Environment strings
-  std::vector<std::string> envStrings = {"OMP_NUM_THREADS=1"};
-  for (std::string& env : envStrings) {
-    for (int i = 0; i < env.size(); i++) {
-      stringBytes.push_back(env.c_str()[i]);
+  for (size_t i = 0; i < Config::get()["Environment-Variables"].size(); i++) {
```

Comment:
I'm unsure how this would be done... Given an environment variable could be anything, or even have none present, how would the node checker allow us to ensure it cam be read as a string? Doesn't the 2nd argument have to be the value itself which we cannot know?

Reply:
You just check against an empty vector and it ensures existence and ability to be read as a string. Then the 2nd arg is just an human-readable key for the failed output.

Reply:
Check added